azure-pipelines.yml

```diff
@@ -57,7 +55,14 @@ phases:
           _BuildConfig: Debug
         release:
           _BuildConfig: Release
-          _DotNetPublishToBlobFeed: false
+          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), contains(variables['Build.DefinitionName'], 'official')) }}:
+            _DotNetPublishToBlobFeed: true
+            _PublishArgs: /p:PublishToSymbolServer=true
+              /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
+              /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
+              /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
+              /p:DotNetPublishBlobFeedUrl=$(_PublishBlobFeedUrl)
+              /p:DotNetPublishToBlobFeed=$(_DotNetPublishToBlobFeed)
```

Comment:
I moved this here so that we could avoid doing all of this junk for non-release stuff. This seems to be the simplest way to do things.